### PR TITLE
New Version: Canonical.Ubuntu.2204 2204.1.7.0

### PR DESCRIPTION
**Pull request by BrandonWanHuanSheng**
Newer Version Canonical.Ubuntu.2204 2204.1.7.0

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102697)

### CODE REVIEW

File: manifests/c/Canonical/Ubuntu/2204/2204.1.7.0/Canonical.Ubuntu.2204.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.2.4 $debug=NVS0.CRLF.5-1-19041-2673.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Canonical.Ubuntu.2204
+PackageVersion: 2204.1.7.0
+MinimumOSVersion: 10.0.0.0
```
Comment (russellbanks) [inline manifests/c/Canonical/Ubuntu/2204/2204.1.7.0/Canonical.Ubuntu.2204.installer.yaml]:
```suggestion
MinimumOSVersion: 10.0.16215.0
```

**Review by russellbanks — CHANGES_REQUESTED** [review]

File: manifests/c/Canonical/Ubuntu/2204/2204.1.7.0/Canonical.Ubuntu.2204.installer.yaml (hunk 1 above)
Comment (BrandonWanHuanSheng) [reply manifests/c/Canonical/Ubuntu/2204/2204.1.7.0/Canonical.Ubuntu.2204.installer.yaml]: I already address the issue

**Comment by russellbanks** [comment]
[Policy] reset feedback

**Comment by BrandonWanHuanSheng** [comment]
@russellbanks I always looking at the AzurePipelines.

**Comment by BrandonWanHuanSheng** [comment]
@stephengillie. It look like my PR failed up from the automatic validation.

**Comment by BrandonWanHuanSheng** [comment]
> Hello @BrandonWanHuanSheng,
> 
> One or more of the installer URLs doesn't appear valid.
> 
> This may happen for sites with policies prohibiting distribution or use by third parties.
> 
> This may happen for URLs pointing to domains that do not align with the publisher domain or package domain. If you could provide supporting evidence from the publisher that the URLs for the installer are correct, that would help us to validate and approve this PR.
> 
> Template: msftbot/validationError/urls/domain

It is correct Domain URL. That is no second domain link address.

**Comment by stephengillie** [comment]
> @stephengillie. It look like my PR failed up from the automatic validation.

Hi @BrandonWanHuanSheng,

I can't navigate to the InstallerUrl from the PackageUrl. This appears to be the Windows Store package. How did you locate it?

**Comment by BrandonWanHuanSheng** [comment]
> > @stephengillie. It look like my PR failed up from the automatic validation.
> 
> Hi @BrandonWanHuanSheng,
> 
> I can't navigate to the InstallerUrl from the PackageUrl. This appears to be the Windows Store package. How did you locate it?

This is normally from a documentations, and from the issue to found the download link

**Comment by stephengillie** [comment]
Could you link to where you found the InstallerUrl? We might need to update the PackageUrl to point there.

**Comment by BrandonWanHuanSheng** [comment]
@stephengillie it look like I found the installer URL over here? https://learn.microsoft.com/en-us/windows/wsl/

**Comment by BrandonWanHuanSheng** [comment]
If yes, I will start automatic process 2 change to the manifest. Timer 3 minutes

**Comment by stephengillie** [comment]
> @stephengillie it look like I found the installer URL somewhere over here? https://learn.microsoft.com/en-us/windows/wsl/

Found it - on the [manual install](https://learn.microsoft.com/en-us/windows/wsl/install-manual#step-4---download-the-linux-kernel-update-package) page (first link on the page which you linked), under Step 4.

File: manifests/c/Canonical/Ubuntu/2204/2204.1.7.0/Canonical.Ubuntu.2204.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+PackageIdentifier: Canonical.Ubuntu.2204
+PackageVersion: 2204.1.7.0
+PackageLocale: en-US
+Publisher: Canonical Group Limited
+PublisherUrl: https://canonical.com/
+PublisherSupportUrl: https://ubuntu.com/contact-us
+PrivacyUrl: https://ubuntu.com/wsl
+Author: Canonical Ltd. and the OpenSource community
+PackageName: Ubuntu 22.04 LTS
+PackageUrl: https://learn.microsoft.com/en-us/windows/wsl/
```
Comment (stephengillie) [inline manifests/c/Canonical/Ubuntu/2204/2204.1.7.0/Canonical.Ubuntu.2204.locale.en-US.yaml:13]:
```suggestion
PackageUrl: https://learn.microsoft.com/en-us/windows/wsl/install-manual#step-4---download-the-linux-kernel-update-package
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**